sdk/communication/communication-chat/src/generated/src/chatApiClient.ts

```diff
@@ -38,7 +38,7 @@ export class ChatApiClient extends coreClient.ServiceClient {
       requestContentType: "application/json; charset=utf-8"
     };
 
-    const packageDetails = `azsdk-js-communication-chat/1.3.1`;
+    const packageDetails = `azsdk-js-communication-chat/1.3.1-beta.1`;
```

Comment:
1.3.1 has been GAed already, you might want 1.3.2-beta.1

Reply:
updated